code/API_definitions/network_access_management.yaml

```diff
@@ -0,0 +1,2863 @@
+                BaseServiceSiteList:
+                  $ref: "#/components/examples/BaseServiceSiteList"
+                ServiceSiteListWithLocation:
+                  $ref: "#/components/examples/ServiceSiteListWithLocation"
+                ServiceSiteListWithDevices:
+                  $ref: "#/components/examples/ServiceSiteListWithDevices"
+                ServiceSiteListWithLocationAndDevices:
+                  $ref: "#/components/examples/ServiceSiteListWithLocationAndDevices"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "501":
+          $ref: "#/components/responses/Generic501"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+  /service-sites/{siteId}:
+    get:
+      tags:
+        - Service Sites
+      summary: Get a service site by ID
+      security:
+        - oauth2:
+            - cpe-management-service-sites-read
+      operationId: getServiceSite
+      description: |
+        Returns a service site that matches the given ID.  Scopes impact the output of this endpoint.  See the examples for various permutations.
+
+        **Required Scopes:**
+          - **any of**
+            - `cpe-management-service-sites-read`
+            - `cpe-management-service-sites-location-read`
+
+        **Optional Scopes:**
+          - `cpe-management-devices-read`
+          - `cpe-management-devices-write`
+          - `cpe-management-devices-hardware-address-read`
+      parameters:
+        - $ref: "#/components/parameters/siteId"
+      responses:
+        "200":
+          description: Contains information about the service site
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ServiceSite"
+              examples:
+                BaseServiceSite:
+                  $ref: "#/components/examples/BaseServiceSite"
+                ServiceSiteWithLocation:
+                  $ref: "#/components/examples/ServiceSiteWithLocation"
+                ServiceSiteWithDevices:
+                  $ref: "#/components/examples/ServiceSiteWithDevices"
+                ServiceSiteWithLocationAndDevices:
+                  $ref: "#/components/examples/ServiceSiteWithLocationAndDevices"
+        "400":
+          $ref: "#/components/responses/Generic400"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "404":
+          $ref: "#/components/responses/ServiceSiteNotFound404"
+        "500":
+          $ref: "#/components/responses/Generic500"
+        "501":
+          $ref: "#/components/responses/Generic501"
+        "503":
+          $ref: "#/components/responses/Generic503"
+
+  /service-sites/{siteId}/devices:
+    get:
+      tags:
+        - Service Sites
+        - Devices

```

Comment:
```suggestion
```

Reply:
Using the tag for the resource actually returned, so Devices instead of Service Site.